src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -874,7 +874,7 @@ class $class extends $baseClass
 {
     private \$parameters;
     private \$targetDirs = array();
-    private \$privates = array();
+    protected \$privates = array();
```

Comment:
A small comment should be added to explain why this specific one is not private.